packages/webdriver/src/types.ts

```diff
@@ -306,7 +306,7 @@ export interface VendorExtensions extends EdgeCapabilities {
     // Testingbot w3c specific
     'tb:options'?: TestingbotCapabilities
     // Saucelabs w3c specific
-    'sauce:options'?: SauceLabsCapabilities
+    'sauce:options'?: SauceLabsCapabilities & SauceLabsVisualCapabilities
```

Comment:
Visual options live in a new top-level capability `sauce:visual`. It is parallel to sauce:options